.gitignore

```diff
@@ -25,3 +25,7 @@ Pipfile
 **/__pycache__
 openms-streamlit-vue-component/
 js-component/
+
+
+workspaces-flashtaggerviewer
+dist
```

Comment:
dist should be not ignored

Reply:
Depends :D 

The way I built it, dist is created during the building of the Docker Container. I think this may be more convenient as it ensures the Vue-Component is always at the current version without manually having to update it.

Reply:
I agree that it's more convenient for maintenance purposes, but this seems like FLASHViewer is assumed to be executed only through Docker. Maybe we want to open the possibility that FLASHViewer can be run locally, and not through Windows executable files as well. I believe building the Docker container is similar to "release" and we want to actively update FLASHViewer which will not include Docker

Reply:
That makes sense. I removed it from the exclusion list.